cli/src/cmd/forge/test.rs

```diff
@@ -43,31 +43,63 @@ pub struct Filter {
     pub pattern: Option<regex::Regex>,
 
     /// Only run test functions matching the specified regex pattern.
-    #[clap(long = "match-test", alias = "mt", conflicts_with = "pattern")]
+    #[clap(
+        long = "match-test",
+        alias = "mt",
+        conflicts_with = "pattern",
+        value_name = "REGEX",
+        help = "Only run test functions matching the specified regex pattern"
```

Comment:
we should only have this in one place, having this as docs _and_ as an attribute is redundant.
this is also currently not consistent across the whole codebase, I personally prefer the help attribute because that makes it obvious.

so let's remove the redundant comments then?